ui/components/MeshConfigComponent.js

```diff
@@ -132,6 +132,12 @@ function MesherySettingsNew({ classes, enqueueSnackbar, closeSnackbar, updatePro
               id : ctx.id
             };
             tableInfo.push(data);
+            fetchMesheryOperatorStatus({ k8scontextID : ctx.id }).subscribe({
+              next : (res) => {
+                setOperatorState(res, index);
+              },
+              error : (err) => console.log("error at operator scan: " + err),
+            });
```

Comment:
If you want to get an instant information of the actual state present in operator, you should just query it once, and instead rely on global subscriptions for any change reflected in settings page.

Reply:
I am not using a subscription.
I added query in toggleOperatorState because sometimes, redeploy fails even then switch position is off, and operator subscription is sometimes failing so correct status is not shown.